### PR TITLE
Re-add Veeam Listing workaround

### PR DESCRIPTION
**Pull request by klauspost**
## Description

Re-add #10893 - this time a bit simpler.

## Motivation and Context

Same as last time

## Types of changes
- [x] Optimization (provides speedup with no functional changes)

### CODE REVIEW

File: cmd/erasure-server-pool.go [hunk 1]
```diff
@@ -33,7 +33,7 @@ import (
 
 	"github.com/dustin/go-humanize"
 	"github.com/google/uuid"
-	"github.com/minio/madmin-go/v2"
+	"github.com/minio/madmin-go"
```
Comment (harshavardhana) [inline cmd/erasure-server-pool.go]:
```suggestion
	"github.com/minio/madmin-go/v2"
```

Still needs to be /v2

**Review by harshavardhana — APPROVED** [review]

File: cmd/erasure-server-pool.go (hunk 1 above)
Comment (klauspost) [reply cmd/erasure-server-pool.go]: Yeah not sure why goimports sometimes messes this up.

**Review by vadmeste — APPROVED** [review]
LGTM

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-erasure.sh | :heavy_check_mark: |
| mint-dist-erasure.sh | :heavy_check_mark: |
| mint-compress-encrypt-dist-erasure.sh | :heavy_check_mark: |
| mint-pools.sh | :heavy_check_mark: |
Deleting image on docker hub
Deleting image locally

**Pull request merged**